styles/main.css

```diff
@@ -134,10 +135,9 @@ figure img {
 .favourite {
     clear:both;
 }
-.back-to-top a{
-    position:relative;
-    bottom: 0.5em;
-    right: -1em;
+.back-to-top a {
+    position:fixed;
+    bottom: 3.5em;

```

Comment:
You want `position: fixed` here to make it be positioned relative to the browser window.
Leaving `right` unspecified makes it so it is positioned horizontally at its normal flow position, which is at hte right side of the `.back-to-top` element due to the `text-align: right` declaration